modules/platforms/build.gradle

```diff
@@ -19,6 +19,7 @@ plugins {
     id 'signing'
     alias(libs.plugins.checksum)
     alias(libs.plugins.cmake)
+    id 'com.bmuschko.docker-remote-api' version '9.4.0'
```

Comment:
Move plugin version to `libs.yml` under `[plugins]` section.  Then add the import into root `build.gradle` with `apply false`. After that you are free to drop `version '9.4.0'` here.

Reply:
Done.